### ISSUE TITLE
Review scheduling and benefits of Team and cohort SGC

### ISSUE THREAD

**Issue by ByteSizedIT** [foundersandcoders/coursebook#828]
Raised during week 5 SGC by FAC26. Suggested that there is a lot of repetition and questioned the value of some of the points raised/discussed

**Comment by ByteSizedIT** [comment]
Anna facilitated a Feedback session. Awaiting results of feedback survey on the session before reviewing the timetable moving forward.

Action: @ByteSizedIT to meet with Anna to follow up

**Comment by ByteSizedIT** [comment]
We collected feedback from FAC26 on the Feedback session that Anna delivered, as well as their cohort SGCs and team SGCs. The survey responses are [here](https://airtable.com/appKA4hcyDYWp46Mx/tblboPGRkimNR5Rx7/viwNj026ZAcgQFSBd?blocks=hide).

As a result we have reduced the number of Cohort SGC's in the schedule by five, and introduced three 1-1 Feedback Sessions to the schedule:

- An 80min Introduction to 1-1 Feedback, to be facilitated by Anna or the CF. This is scheduled in Week 3 of the full-time course, in place of Cohort SGC and utilising an additional 30mins that was available due to there being no topic/workshops to introduce for week 4. The timing also allows a couple of weeks for Cohort SGC to bed in and for the cohort to get to know each other in person.

- Two 50 min 1-1 Feedback sessions, to be facilitated by the CF. These are scheduled towards the end of each Design week - giving the opportunity for feedback to be given to team members ahead of Build. 

- Two further cohort SGC's have been removed from the schedule - from In-house Build Week 1 and TfB Build Week 1. This time has been made available as Build time.
